tools/networkmanager/inject_txs.go

```diff
@@ -215,4 +216,10 @@ func checkL2TxsSuccessful(ctx context.Context, rpcHandles *network.RPCHandles, t
 	} else {
 		println(fmt.Sprintf("Successfully injected %d withdrawals into the L1.", injectedWithdrawals))
 	}
+
+	if notFoundNativeTransfers != 0 {
```

Comment:
This **if** contains a lot of negatives obfuscating the condition.

Reply:
Yea, made a small change that makes it more readable I think. WDYT ?

Reply:
It took me a bit to interpret notFoundNativeTransfers as a count (even if compared with 0). Given the condition perhaps `hasMissingNativeTransfers` would be slightly better